node-graph/graph-craft/src/graphene_compiler.rs

```diff
@@ -17,15 +17,23 @@ impl Compiler {
 		network.remove_redundant_id_nodes();
 		network.remove_dead_nodes();
 		let proto_networks = network.into_proto_networks();
```

Comment:
```suggestion
		let proto_networks = network.into_proto_networks()?;
```
You should be able to just use the `?` operator and then wrap the proto_networks in an Ok()

Reply:
I  could be wrong but I think_ `proto_networks_result` is the one that returns the Result, so I added the question mark operator to that expression instead of `proto_networks` - the match statement on the bottom then can just be wrapped in Ok() like you said